tests/cross_functional/scale/test_multiple_storage_coexistence.py

```
@@ -1,223 +1,89 @@
         sc_name = helpers.create_unique_resource_name("scale-test", "sc")
         rc_name = helpers.create_unique_resource_name("scale-test", "rc")
         user_secret_name = f"{rc_name}-user-details-secret"
 
-        # Cache dynamic names to the test class instance so the validation methods can reference them
         self.sc_name = sc_name
         self.rc_name = rc_name
         self.user_secret_name = user_secret_name
 
+        tracked_resources = []
+
         def finalizer_cleanup():
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `finalizer_cleanup`

Add return type annotation: `None`

(ANN202)